### ISSUE TITLE
React events not triggering

### ISSUE THREAD

**Issue by narendravenkata** [rstaib/jquery-steps#250]
Hi , I am using rubix template and in the template making use of jquery-steps plugin but my events are not triggerring

**Comment by timotew** [comment]
+1

**Comment by syedolive** [comment]
have you found a solution for this yet ?? its driving me crazy

**Comment by narendravenkata** [comment]
@syedolive  hi , i am not using this jquery-steps. i am using this 
[https://github.com/newbreedofgeek/react-stepzilla]

**Comment by narendravenkata** [comment]
hi , i am not using this jquery-steps. i am using this
https://github.com/newbreedofgeek/react-stepzilla

**Comment by syedolive** [comment]
> hi , i am not using this jquery-steps. i am using this
> https://github.com/newbreedofgeek/react-stepzilla

Yes , have seen this. But I'm limitless templates which i bought from themeforest and they have used this plugin to create wizards which are now a part of our system requirements .
